### DISCUSSION TITLE
reDOS took dosemu2

### DISCUSSION THREAD

**Discussion by stsp** [dosemu2/dosemu2#2250] [General]
Found this by some googling:
https://files.red-soft.ru/redos/8.0/x86_64/updates/dosemu2-2.0pre9-1.git.221.c091ff9.red80.x86_64.rpm
So dosemu2 entered the rpm-based
OS with that interesting name reDOS.
They package the full stack from
nasm-segelf to comcom64. Which is
quite easy, given that I provide the
ready-to-use rpm specs.
A good sign! First distro. :)
Hope debian comes next.
Yes, yes, I am aware about arch.
But in arch, this is a user's repo.

**Comment by jschwartzenberg** [comment]
Peculiar distro! From what I can see, there's no source code btw.

**Reply by stsp** [reply]
Source code of what?
If they package open-source, the
source code is available in either case.

**Reply by jschwartzenberg** [reply]
Maybe all their packages are 100% matching with upstream, but they would still have an obligation to make sure that the sources they used are available. (Though I have no idea whether copyleft would be in any way applicable/enforceable given the context.)

Most distros have a directory structure that does incorporate the source code as part of the hierarchy with downloads. For an RPM-based distro, it would contain all the source RPMs. I never saw that it was simply omitted.

**Comment by stsp** [comment]
Среда, 31 июля 2024 г получено от Julius Schwartzenberg:
> Maybe all their packages are 100% matching with upstream, but they would still have an obligation to make sure that the sources they used are available. (Though I have no idea whether copyleft would be in any way applicable/enforceable given the context.)

URL to upstream should be enough
in that case. Their rpms have an URL.
Maybe by not providing src.rpms they
avoid being repackaged, as centos
repacks rhel.
